app/errors.py

```diff
@@ -0,0 +1,21 @@
+class VaccineError(Exception):
+    # basic class for errors about vaccination
+    pass
+
+class NotVaccinatedError(VaccineError):
+    """Raised when a person is not vaccinated."""
+    def __init__(self, message: str = "Visitor is not vaccinated") -> None:
+        super().__init__(message)
+
+
+class OutdatedVaccineError(VaccineError):
+    """Raised when the vaccine has expired."""
+    def __init__(self, message: str = "Vaccine is outdated") -> None:
+        super().__init__(message)
+
+
+class NotWearingMaskError(Exception):
+    """Raised when a person is not wearing a mask."""
+    def __init__(self, message: str = "Visitor is not wearing a mask"
+                 ) -> None:
```

Comment:
The constructor of `NotWearingMaskError` is split across two lines in a way that can be improved for readability. Consider placing the closing parenthesis on the same line as the message string or aligning it properly.